mkdocs.yml

```diff
@@ -36,15 +36,18 @@ nav:
       - Models: 'concepts/models_explainer.md'
       - Splitters: 'concepts/splitters.md'
     - Examples:
-      - Technical Walkthroughs: 'examples/technical.md'
-      - Example Cookbook: 
-        - Date-time and Holiday features: 'examples/date'
-        - Ensembling models: 'examples/ensembling'
-        - Preprocess your data: 'examples/preprocess'
-        - Tabular ML models: 'examples/tabular'
-        - Transform your target variable: 'examples/transform-target'
+      - Walkthroughs: 
```

Comment:
a quick question: why don't we separate out examples from walkthroughs?
in my mind, they're quite different, walkthrough has a lot of explanation, while examples are short, and without any comment

Reply:
I'm easy either way!

Reply:
Separated it out.